keps/sig-node/1769-memory-manager/README.md

```diff
@@ -521,54 +586,61 @@ _This section must be completed when targeting alpha to a release._
 
-* **Does enabling the feature change any default behavior?**
+###### Does enabling the feature change any default behavior?
+
 Yes, the admission flow changes for a pod in Guaranteed QoS class. With the Memory Manager, Topology Manager also takes into account allocatable memory (and hugepages) to either admit a pod to the node or reject it. 
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes, it uses a feature gate.
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, it uses a feature gate.
 
   <!-- Also set `disable-supported` to `true` or `false` in `kep.yaml`.
   Describe the consequences on existing workloads (e.g., if this is a runtime
   feature, can it break the existing applications?). -->
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+###### What happens if we reenable the feature if it was previously rolled back?
+
 The Memory Manager utilizes the state file to track memory assignments. If State file is not valid, it must be removed and kubelet restarted. E.g., State file might become invalid when kube/system reserved have changed (increased), which may lead to a situation when some containers cannot be started.
 
-* **Are there any tests for feature enablement/disablement?**
+###### Are there any tests for feature enablement/disablement?
+
 Yes, there is a number of Unit Tests designated for State file validation.
 
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
-* **How can a rollout fail? Can it impact already running workloads?**
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
   It is possible that the state file will have inconsistent data during the rollout, because of the kubelet restart, but
   you can easily to fix it by removing memory manager state file and run kubelet restart. It should not affect any running 
   workloads.
-  
 
-* **What specific metrics should inform a rollback?**
+
+###### What specific metrics should inform a rollback?
+
   The pod may fail with the admission error because the kubelet can not provide all resources aligned from the same NUMA node. 
   You can see the error message under the pod events.
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
   Tested it manually by replacing the kubelet binary on the node with the `Static` memory manager policy, but I failed
   to find correct procedure how to test upgrade from 1.21 to my custom build with updated kubelet binary.
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
```

Comment:
We have some trailing text from previous sentence (from the older template) below, can we remove that?